### PR TITLE
Docs/229/remove refactor as breaking change example

### LINKED ISSUE #229: refactor example isn't a refactor
The [example of a refactor](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-both-and-breaking-change-footer) is

> refactor!: drop support for Node 6
>
> BREAKING CHANGE: refactor to use JavaScript features not available in Node 6.

But is this really a refactor? I guess people have slightly different definitions of refactoring but Wikipedia says (emphasis mine):

> Code refactoring is the process of restructuring existing computer code—changing the factoring—**without changing its external behavior**

With that definition you cannot have a refactor commit that also contains a breaking change.

I think the example should be changed to something like:

> refactor: extract foo function to reduce duplication

The example containing `!` could be changed to a `feat:`.

### PR DESCRIPTION
**Pull request by shaman-apprentice**
Close #229 

Also see #229 for further reasoning, which basically says the same as related commit messages so far.

### CODE REVIEW

**Comment by shaman-apprentice** [comment]
Scanning the PRs I realised, that https://github.com/conventional-commits/conventionalcommits.org/pull/406/commits/dba37abd734f1c5b48234dda7fd25ce8379561c7 would make #337 obsolete as well, if you agree it is a good idea. But I am not too hard opionated about it and we could also take this commit out of the PR.

**Comment by shaman-apprentice** [comment]
It's been a month without any negative feedback. I think having something other than a refactoring as example for a breaking change takes away one opinionated pitfall. Therefore I would appreciate a merge :)

File: content/next/index.md [hunk 1]
```diff
@@ -53,19 +53,12 @@ BREAKING CHANGE: `extends` key in config file is now used for extending other co
...
-feat!: drop support for Node 6
+feat!: enforce authentication for all requests
 ```
 
 ### Commit message with scope and `!` to draw attention to breaking change
 ```
-refactor(runtime)!: drop support for Node 6
-```
-
-### Commit message with both `!` and BREAKING CHANGE footer
```
Comment (damianopetrungaro) [inline content/next/index.md:64]: I'd still keep both the example with both of them

File: content/next/index.md [hunk 2]
```diff
@@ -53,19 +53,12 @@ BREAKING CHANGE: `extends` key in config file is now used for extending other co
 
 ### Commit message with `!` to draw attention to breaking change
 ```
-feat!: drop support for Node 6
+feat!: enforce authentication for all requests
```
Comment (damianopetrungaro) [inline content/next/index.md:56]: What about a more product oriented example?

`feat!: send an email to the customer when the product is shipped`

File: content/next/index.md (hunk 1 above)
Comment (shaman-apprentice) [reply content/next/index.md:64]: I dropped my first commit again and replaced the `refactor(runtime)!` through a `chore(runtime)!`. I think `chore` is the correct scope for dropping node support, as it is neither really a `feat` nor a `refactor` in my opinion.

File: content/next/index.md (hunk 2 above)
Comment (shaman-apprentice) [reply content/next/index.md]: cool, I like your example. I updated to `send an email to the customer when a product is shipped` (I felt free to replace "the" through "a").

Comment (shaman-apprentice) [reply content/next/index.md]: (and of course thanks a lot for your review, your time and energy for this whole project! ❤️ )

**Comment by damianopetrungaro** [comment]
Thanks @shaman-apprentice and sorry for the late reply, busy days 😄

**Pull request merged**